db/migrate/20231128103759_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :content
```

Comment:
- [ ] Kindly update your migration for the `Comment` table to match the requirement [diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png) as the table contains the `text` column instead of `content` column.

Reply:
Thanks for identifying the issue has been resolved 😃